### PR TITLE
Staff Training

### PR DESCRIPTION
**Pull request by ChazUK**
Add / View / Edit / Delete Training records for staff.

It also splits the Worker Summary into individual components that can be reused. The Summary and the Check Changes screen are now separate so that one can be used to Confirm and Save the Staff Record, and the other can be used to view the basic record and qualifications and training information on separate tabs.

### CODE REVIEW

**Comment by ChazUK** [comment]
[Add/ Update worker training #88](https://trello.com/c/t2xFhyaa/341-add-update-worker-training-88)

**Comment by ChazUK** [comment]
[Change Request - Don't know option on "Training Accredited"](https://trello.com/c/uurKN77W/419-change-request-dont-know-option-on-training-accredited)

File: src/app/features/workers/add-edit-training/add-edit-training.component.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+          <p>Enter a training name</p>
+        </div>
+        <div *ngIf="submitted && form.get('tile').hasError('maxlength')" class="alert alert-danger">
+          <p>Training name must be no longer than {{ form.get('title').getError('maxlength').requiredLength }} characters.</p>
+        </div>
+        <input type="text" formControlName="title" class="form-control col-12" maxlength="120" />
+      </div>
+
+      <div class="field-group mb-5">
+        <label for="accredited" class="d-block">
```
Comment (omarCreativeDev) [inline src/app/features/workers/add-edit-training/add-edit-training.component.html]: isn't d-block a bootstrap class? i think bootstrap has been removed

Comment (ChazUK) [reply src/app/features/workers/add-edit-training/add-edit-training.component.html]: Bootstrap is in the current project but is being removed by your GovUK Frontend Framework stuff

**Pull request merged**